specifications/api/swagger.yaml

```diff
@@ -136,7 +136,7 @@ x-byronWalletMnemonicSentence: &byronWalletMnemonicSentence
     format: bip-0039-mnemonic-word{english}
   example: [
     "squirrel", "material", "silly", "twice", "direct", "slush",
-    "pistol", "razor", "become", "junk", "kingdom", "flee"]
+    "pistol", "razor", "become", "junk", "kingdom", "junk"]
```

Comment:
What's the rationale here?

Reply:
The rationale is not to point to any existing / possibly existing wallet. We had that also as a requirement for new wallet due to historical reasons (afaik the mnemonics in spec for old wallet where sometimes source of confusion for the users. :man_shrugging:)